src/main/java/model/Cars.java

```diff
@@ -0,0 +1,63 @@
+package model;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars (List<Car> cars) {
```

Comment:
```suggestion
    public Cars(List<Car> cars) {
```
개인적으로 메소드명 또는 생성자와 파라미터 사이에 공백을 사용하지 않는 편입니다 😄